testing/citest/tests/openstack_smoke_test.py

```diff
@@ -152,6 +167,18 @@ def create_a_security_group(self):
     (builder.new_clause_builder(
         'Security Group Created', retryable_for_secs=30)
      .show_resource('security group', self.TEST_SECURITY_GROUP)
+<<<<<<< HEAD
+     .contains_pred_list([jp.DICT_MATCHES({'name': jp.STR_SUBSTR(self.TEST_SECURITY_GROUP),
+                      'rules': jp.STR_SUBSTR("protocol='tcp'")
+                               and jp.STR_SUBSTR("port_range_min='80'")
```

Comment:
could you indent this so it is clearer. I think like this:
.contains_pred_list([
.....jp.DICT_MATCHES({
..........'key': value,
.....}),
.....jp.DICT_MATCHES({
.....})
])